suites/upgrade/client-upgrade/infernalis-client-x/basic/0-cluster/start.yaml

```diff
@@ -0,0 +1,14 @@
+roles:
+- - mon.a
+  - mds.a
+  - osd.0
+  - osd.1
+- - mon.b
```

Comment:
Minor: thinking about our current limited testing resources, it might be nice to have all client-upgrade tests only require two nodes: one for the OSDs/MONs and the other for the client.